src/linnia.js

```diff
@@ -10,6 +10,8 @@ import _recordsFunctions from './records';
 import _permissionsFunctions from './permissions';
 import _util from './util';
 
+const StowAdresses = require('@stowprotocol/stow-addresses')
```

Comment:
Make this an import and group it with the ones above it plz

Reply:
done